CHANGELOG.md

```diff
@@ -1,230 +1,9 @@
 # Azure Mobile Services Change Log
 
-### iOS SDK: Version 2.0
-- GA of offline sync changes from previous betas
+## [iOS SDK](CHANGELOG.ios.md)
```

Comment:
We should put these links on the readme.md in the changelog section and have it no longer point to this changelog.